lib/modules/manager/gradle/artifacts.spec.ts

```diff
@@ -0,0 +1,333 @@
+import os from 'os';
+import { join } from 'upath';
+import {
+  envMock,
+  mockExecAll,
+  mockExecSequence,
+} from '../../../../test/exec-util';
+import {
+  env,
+  fs,
+  git,
+  logger,
+  mockedFunction,
+  partial,
+} from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
+import { TEMPORARY_ERROR } from '../../../constants/error-messages';
+import { resetPrefetchedImages } from '../../../util/exec/docker';
+import { ExecError } from '../../../util/exec/exec-error';
+import type { StatusResult } from '../../../util/git/types';
+import { getPkgReleases } from '../../datasource';
+import { updateArtifacts } from '.';
+
+const platform = jest.spyOn(os, 'platform');
+jest.mock('../../../util/fs');
+jest.mock('../../../util/git');
+jest.mock('../../../util/exec/env');
+jest.mock('../../datasource');
+
+process.env.BUILDPACK = 'true';
+
+const adminConfig: RepoGlobalConfig = {
+  // `join` fixes Windows CI
+  localDir: join('/tmp/github/some/repo'),
+  cacheDir: join('/tmp/cache'),
+  containerbaseDir: join('/tmp/cache/containerbase'),
+};
+
+jest.spyOn(os, 'platform').mockReturnValue('linux');
```

Comment:
```suggestion
platform.mockReturnValue('linux');
```
don't spy twice 😉

can probably be removed, because you mock again in beforEach